app/dashboard/views.py

```diff
@@ -962,8 +961,15 @@ def toolbox(request):
             'link_copy': _('Try It'),
             "active": "true",
             'stat_graph': 'gas_page',
-            },
-           {
+            }, {
```

Comment:
👍

Reply:
Where are the tools portion defined now?